asserts/asserts.go

```diff
@@ -146,6 +158,20 @@ func (ab *assertionBase) Signature() (content, signature []byte) {
 	return ab.content, ab.signature
 }
 
+// SigningKey returns the signer identifier and key id for the key that signed this assertion.
+func (ab *assertionBase) SigningKey() (signedID, keyID string, err error) {
```

Comment:
Does it make sense to decode the signature earlier (in Assemble perhaps) and assume this will always succeed to remove `error` from the result?